pyadjoint/tape.py

```diff
@@ -312,12 +312,23 @@ def enable_checkpointing(self, schedule):
         Args:
             schedule (checkpoint_schedules.schedule): A schedule provided by the
             checkpoint_schedules package.
+            gc_timestep_frequency (int): The timestep frequency for garbage collection.
```

Comment:
Same docstring comment as above.

Also, why is this being repeated? It might make more sense to say: "for more information about this argument refer to :class:`CheckpointManager`"